cime_config/testdefs/testmods_dirs/clm/collapse_nat_pfts_decStart_f10/user_nl_clm

```diff
@@ -0,0 +1,2 @@
+n_dom_pfts = 2
```

Comment:
The name of this test directory, doesn't need the resolution in it. There's nothing in this test that is resolution dependent, hence the resolution shouldn't be in the name.

Reply:
Renamed this test.